### PR TITLE
add hardware-exporter

### PR DESCRIPTION
**Pull request by matthewei**
Hi,
I want to add the hardware-exporter so that someone can use it to deploy it . please kind solve it. many thanks!

### CODE REVIEW

File: charts/hardware-exporter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+annotations:
+  category: Analytics
+  licenses: Apache-2.0
+apiVersion: v2
+appVersion: 1.5.0
```
Comment (imlense) [inline charts/hardware-exporter/Chart.yaml:5]: add home and icon keywords

File: charts/hardware-exporter/values.yaml [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    sourceLabels:
+    - __meta_kubernetes_pod_node_name
+    targetLabel: instance
+  ## @param serviceMonitor.metricRelabelings MetricRelabelConfigs to apply to samples before ingestion
+  ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#relabelconfig
+  ##
+  metricRelabelings: []
+  ## @param serviceMonitor.labels Extra labels for the ServiceMonitor
+  ##
+  labels: {}
```
Comment (imlense) [inline charts/hardware-exporter/values.yaml:325]: add release: metrics default labels

File: charts/hardware-exporter/values.yaml [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+  ##    memory: 128Mi
+  limits: {}
+  ## Examples:
+  ## requests:
+  ##    cpu: 100m
+  ##    memory: 128Mi
+  requests: {}
+## @param podLabels Pod labels
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+##
```
Comment (dongjiang1989) [inline charts/hardware-exporter/values.yaml:211]: add livenessProbe and readinessProbe

**Review by matthewei — COMMENTED** [review]
yes, I will follow your suggestion, Thanks a lot.

File: charts/hardware-exporter/README.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+| `diagnosticMode.enabled` | Enable diagnostic mode (all probes will be disabled and the command will be overridden)      | `false`         |
+| `diagnosticMode.command` | Command to override all containers in the the deployment(s)/statefulset(s)                   | `["sleep"]`     |
+| `diagnosticMode.args`    | Args to override all containers in the the deployment(s)/statefulset(s)                      | `["infinity"]`  |
+
+### Node Exporter parameters
+
+| Name                                          | Description                                                                                                                                                        | Value                   |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------- |
+| `hostAliases`                                 | Deployment pod host aliases                                                                                                                                        | `[]`                    |
```
Comment (matthewei) [inline charts/hardware-exporter/README.md:80]: Please add your readme.

**Comment by matthewei** [comment]
I want to close this

**Pull request closed without merging**